frontend/src/metabase/auth/actions.ts

```diff
@@ -13,6 +16,16 @@ import {
 } from "./analytics";
 import { LoginData } from "./types";
 
+export const REFRESH_LOCALE = "metabase/user/REFRESH_LOCALE";
+export const refreshLocale = createThunkAction(
+  REFRESH_LOCALE,
+  () => async (dispatch: any, getState: () => State) => {
+    const userLocale = getState().currentUser?.locale;
```

Comment:
nip: we can use [`getUser` selector here](https://github.com/metabase/metabase/blob/d6c78de7620b728bfb6b44f2ac74ea713931c5d7/frontend/src/metabase/selectors/user.js#L4)

Reply:
Fixed